api/types/mount/mount.go

```diff
@@ -56,3 +62,24 @@ type Driver struct {
 	Name    string            `json:",omitempty"`
 	Options map[string]string `json:",omitempty"`
 }
+
+// TmpfsOptions defines options specific to mounts of type "tmpfs".
+type TmpfsOptions struct {
+	// Size sets the size of the tmpfs, in bytes.
+	//
+	// This will be converted to an operating system specific value
+	// depending on the host. For example, on linux, it will be convered to
+	// use a 'k', 'm' or 'g' syntax. BSD, though not widely supported with
+	// docker, uses a straight byte value.
+	//
+	// Percentages are not supported.
+	SizeBytes int64
+	// Mode of the tmpfs upon creation
+	Mode os.FileMode
+
+	UID          uint32
```

Comment:
How can any of this possibly be supported if we don't have support in swarmkit? https://github.com/docker/swarmkit/blob/master/api/types.proto#L191